test/e2e/tests/test_replicationgroup.py

```diff
@@ -154,6 +154,24 @@ def rg_cmd_update(rg_cmd_update_input, make_replication_group, rg_deletion_waite
     rg_deletion_waiter.wait(ReplicationGroupId=input_dict["RG_ID"])
 
 
+@pytest.fixture(scope="module")
+def rg_delete_input(make_rg_name):
+    return {
+        "RG_ID": make_rg_name("rg-delete"),
+        "ENGINE_VERSION": "6.x",
+        "NUM_NODE_GROUPS": "1",
+        "REPLICAS_PER_NODE_GROUP": "1"
+    }
+
+
+@pytest.fixture(scope="module")
+def rg_delete(rg_delete_input, make_replication_group):
```

Comment:
this function doesn't delete anything...

Reply:
Ok, changed it to the noun form "deletion" to avoid further confusion